src/query.jl

```diff
@@ -177,8 +178,8 @@ function partial_signature_matching(f::Function, sig::Tuple)
 end
 
 """
-Search loaded documentation for all methods of a generic function `f` that match
-the provided signature `sig`. Optionally, provide an index (1-based) to view an
+Search for all methods of a generic function `f` that match the signature `sig`.
+Optionally, provide an index (1-based) to view an
```

Comment:
The "loaded documentation" is important here since users might assume that Lexicon will search through all docstrings even if their respective module hasn't been `using ModuleName` yet.